.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 *.txt
 test.js
 coverage
+play.js
```

Comment:
why is this added to the gitignore file? if you just used this file to test out the library, please remove it from the pull request.